docs/manage/ml/train-model.md

```diff
@@ -14,13 +14,14 @@ You can tag [images collected](../../../services/data/configure-data-capture/) b
 
 {{<youtube embed_url="https://www.youtube-nocookie.com/embed/CP14LR0Pq64">}}
 
-## Label a dataset
+## Train a model
 
-To label a dataset, go to the [**DATA** tab](https://app.viam.com/data/view) in the Viam app.
+After [labeling your images](/manage/data/label/), click on the **TRAIN MODEL** button in the top right corner.
 
-![Add new model menu](../img/add-new-model.png)
+![Train model button](../img/train-model.png)
 
-On the **Images** subtab, you can filter available images, using the **Filtering** menu and select the attributes that match where, how, and when the data was collected.
+A **Training** side menu opens.
+The model that you configure to be training will train on all images that are part of the current filter.
```

Comment:
[nit] Sentence is kinda awk but I don't have the requisite knowledge to provide a good suggestion. Fine if there isn't a better way.

Reply:
```suggestion
The model will train on all images that are part of the current filter.
```